src/plugins/intel_gpu/src/graph/program_node.cpp

```diff
@@ -1599,11 +1615,11 @@ void program_node::init_onednn_primitive_attributes() {
                                 post_ops.append_eltwise(dnnl::algorithm::eltwise_linear, q_param->_out_scale, 0.0f);
                                 update_onednn_post_op_list(onednn_post_op_type::eltwise_linear, empty_mem);
                             } else {
-                                auto out_scale = get_dependency(dep_idx++).get_output_layout();
+                                auto out_scale = get_input_layout(dep_idx++);
                                 dnnl::memory::desc out_scale_desc = onednn::layout_to_memory_desc(out_scale, dnnl::memory::format_tag::ab, true);
                                 post_ops.append_binary(dnnl::algorithm::binary_mul, out_scale_desc);
                                 update_onednn_post_op_list(onednn_post_op_type::binary_mul, dep_idx - 1, dnnl::memory::format_tag::ab, true,
-                                                           out_scale_desc.get_dims(), out_scale_desc.get_data_type());
+                                                            out_scale_desc.get_dims(), out_scale_desc.get_data_type());
```

Comment:
No need to change it. Please update it too?

Reply:
I do not modify it manually but I rollback it.